ext/civicrm_admin_ui/managed/SavedSearch_Personal_Campaign_Pages.mgd.php

```diff
@@ -0,0 +1,284 @@
+          'description' => NULL,
+          'sort' => [],
+          'limit' => 50,
+          'pager' => [],
+          'placeholder' => 5,
+          'columns' => [
+            [
+              'type' => 'field',
+              'key' => 'title',
+              'dataType' => 'String',
+              'label' => E::ts('Page Title'),
+              'sortable' => TRUE,
+              'link' => [
+                'path' => 'civicrm/pcp/info/?reset=1&id=[id]',
+                'entity' => '',
+                'action' => '',
+                'join' => '',
+                'target' => '',
+              ],
+            ],
+            [
+              'type' => 'field',
+              'key' => 'PCP_Contact_contact_id_01.sort_name',
+              'dataType' => 'String',
+              'label' => E::ts('Supporter'),
+              'sortable' => TRUE,
+              'link' => [
+                'path' => '',
+                'entity' => 'Contact',
+                'action' => 'view',
+                'join' => 'PCP_Contact_contact_id_01',
+                'target' => '',
+              ],
+              'title' => E::ts('View Personal Campaign Page Contact'),
+            ],
+            [
+              'type' => 'field',
+              'key' => 'IFNULL_PCP_ContributionPage_page_id_01_title_PCP_Event_page_id_01_title',
+              'dataType' => 'String',
+              'label' => E::ts('Contribution Page/Event'),
+              'sortable' => TRUE,
+              'rewrite' => '',
+            ],
+            [
+              'type' => 'field',
+              'key' => 'PCP_ContributionPage_page_id_01.start_date',
+              'dataType' => 'Timestamp',
+              'label' => E::ts('Start Date'),
+              'sortable' => TRUE,
+            ],
+            [
+              'type' => 'field',
+              'key' => 'PCP_ContributionPage_page_id_01.end_date',
+              'dataType' => 'Timestamp',
+              'label' => E::ts('End Date'),
+              'sortable' => TRUE,
+              'empty_value' => '(ongoing)',
+            ],
+            [
+              'type' => 'field',
+              'key' => 'status_id:label',
+              'dataType' => 'Integer',
+              'label' => E::ts('Status'),
+              'sortable' => TRUE,
+            ],
+            [
+              'text' => '',
+              'style' => 'default',
+              'size' => 'btn-xs',
+              'icon' => 'fa-bars',
+              'links' => [
+                [
+                  'task' => '',
+                  'entity' => 'PCP',
+                  'join' => '',
+                  'target' => '_blank',
+                  'icon' => 'fa-pencil',
+                  'text' => E::ts('Edit'),
+                  'style' => 'default',
+                  'path' => '',
+                  'action' => 'update',
+                  'condition' => [],
+                ],
+                [
+                  'path' => 'civicrm/admin/pcp?action=revert&id=[id]',
```

Comment:
This won't work because it has the same path as the page.

Reply:
@colemanw Thanks.  That makes sense.  Do we need to create a new page to handle reject/approve?

Reply:
I think we briefly discussed yesterday the idea of making it a bulk action. That would involve adding a couple entity-specific api actions and then implementing `hook_civicrm_searchKitTasks`. Is that something you'd like to take a shot at or should I?

Reply:
@colemanw I'd like to give it a try - Just not sure when I can get it done

Reply:
Ok if you need any help let me know @kcristiano